torch/_dynamo/variables/functions.py

```diff
@@ -1039,6 +1049,83 @@ def call_grid(self, grid, meta, tx):
         grid = grid.call_function(tx, [meta], {})
         return grid
 
+    # We use this function to wrap user-defined functions (e.g., early_config_prune, perf_model)
+    def call_user_defined_fn(self, user_fn, args, kwargs, tx):
+        result = UserFunctionVariable(user_fn).call_function(tx, args, kwargs)
+        return result
```

Comment:
So it seems that `call_user_defined_fn` is only called from the dynamo-specific implementation of  `call_prune_configs_if_required`. Do we really need to make it an abstract method of `TritonHOPifier`? Or maybe a private helper in the `DynamoTritonHOPifier` suffice?